tests/testthat/helper-snapshot.R

```diff
@@ -0,0 +1,22 @@
+expect_snapshot <- function(x, ...) {
+  renv_scope_options(renv.verbose = TRUE)
+
+  # This shouldn't be so hard
+  eval.parent(
+    substitute(
+      testthat::expect_snapshot(x, ..., transform = strip_dirs)
+    )
+  )
+}
+
+strip_dirs <- function(x) {
+  x <- gsub(getwd(), "<wd>", x, fixed = TRUE)
```

Comment:
The amount of stuff accumulating here makes me wonder if there might be a better alternative approach.